### PR TITLE
Modified the login form to match Zepelin's specification. Modified CS…

### PR DESCRIPTION
**Pull request by arturoruiz**
…S properties to match the styling.

### CODE REVIEW

File: frontend/src/core/components/AuthSidebar/AuthSidebar.scss [hunk 1]
```diff
@@ -2,18 +2,37 @@
   margin-top: 35%;
 
   h1 {
-    color: #ffffff;
+    width: 254px;
+    height: 90px;
+    font-family: OpenSans;
```
Comment (xerp) [inline frontend/src/core/components/AuthSidebar/AuthSidebar.scss]: About this properties like font we have a scss file, that has font properties, please import that file(Variables/Font.scss) and use it here.

File: frontend/src/core/components/AuthSidebar/AuthSidebar.scss [hunk 2]
```diff
@@ -2,18 +2,37 @@
...
-    font-weight: 700;
+    letter-spacing: normal;
+    color: #ffffff;
   }
 
   p {
     font-size: 14px;
     letter-spacing: -0.38px;
+    color: #ffffff;
+    font-family: Lato;
```
Comment (xerp) [inline frontend/src/core/components/AuthSidebar/AuthSidebar.scss]: About this properties like font we have a scss file, that has font properties, please import that file(Variables/Font.scss) and use it here.

File: frontend/src/views/Auth/Login/Login.scss [hunk 3]
```diff
@@ -1 +1,40 @@
+.center {
+  padding: 0rem 1rem !important;
+}
 
+h2 {
+  width: 93px;
+  height: 38px;
+  font-family: OpenSans;
```
Comment (xerp) [inline frontend/src/views/Auth/Login/Login.scss]: About this properties like font we have a scss file, that has font properties, please import that file(Variables/Font.scss) and use it here.

**Review by xerp — CHANGES_REQUESTED** [review]

File: frontend/src/views/Auth/Login/Login.scss [hunk 4]
```diff
@@ -1 +1,40 @@
+.center {
+  padding: 0rem 1rem !important;
+}
 
+h2 {
+  width: 93px;
```
Comment (rafaeldeleonp) [inline frontend/src/views/Auth/Login/Login.scss]: Remove this styles. This is being applied to every h2. If you log in and get to the home (Network Status) you will that all h2 titles, the text will be two lines.

**Review by rafaeldeleonp — APPROVED** [review]

**Review by xerp — APPROVED** [review]

**Pull request merged**